Taskfile.yml

```diff
@@ -3,26 +3,34 @@
 version: '3'
 
 vars:
-  DEBUG_LOG: debug.log
-  OPSLEVEL_GO_PKG: "github.com/opslevel/opslevel-go/v2024"
   PLUGIN_DIR: "${HOME}/.terraform.d/plugins/registry.terraform.io/opslevel/opslevel"
-  # 'VERSION_DYNAMIC' needs to run in this scope before 'task terraform-build' begins
-  VERSION_DYNAMIC:
-    sh: echo $(git describe --abbrev=0 | tr -d v)5
   TEST_DIR: "tests"
   WORKSPACE_DIR: "workspace"
 
+includes:
+  debug:
+    taskfile: ./taskfiles/Debug.yml
+    internal: true
+  go:
+    taskfile: ./taskfiles/Golang.yml
+    internal: true
+  install:
+    taskfile: ./taskfiles/Install.yml
+    internal: true
+  terraform:
+    taskfile: ./taskfiles/Terraform.yml
+    internal: true
+
 tasks:
   ci:
     desc: Workflow to run in CI
     deps:
-      - brew-install-jq
-      - install-gofumpt
-      - install-golangci-lint
-      # - install-nilaway
+      - install:brew-install-jq
+      - install:go-install-gofumpt
+      - install:go-install-golangci-lint
     cmds:
-      - task: workspace
-      - task: has-latest-opslevel-go
+      - task: go:workspace
+      - task: go:has-latest-opslevel-go
```

Comment:
Can we remove this check from CI.  I strongly feel that we don't need to be running this as part of the CI process.  If you want to throw this as a catch on release so we don't release without the latest (But that eventually breaks down with hotfix branches)

this check for `has-latest-opslevel-go` is really something only the developer should be doing.